sopel/modules/url.py

```diff
@@ -92,33 +89,13 @@ def setup(bot):
     if not bot.memory.contains('last_seen_url'):
         bot.memory['last_seen_url'] = tools.SopelMemory()
 
-    def find_func(text, clean=False):
-        def trim_url(url):
-            # clean trailing sentence- or clause-ending punctuation
-            while url[-1] in '.,?!\'":;':
-                url = url[:-1]
-
-            # clean unmatched parentheses/braces/brackets
-            for (opener, closer) in [('(', ')'), ('[', ']'), ('{', '}'), ('<', '>')]:
-                if (url[-1] == closer) and (url.count(opener) < url.count(closer)):
-                    url = url[:-1]
-
-            return url
+    # Initialize shortened_urls as a dict if it doesn't exist.
+    if not bot.memory.contains('shortened_urls'):
+        bot.memory['shortened_urls'] = tools.SopelMemory()
 
-        re_url = r'(?u)((?<!%s)(?:http|https|ftp)(?::\/\/\S+))'\
-            % (bot.config.url.exclusion_char)
-        r = re.compile(re_url, re.IGNORECASE)
 
-        urls = re.findall(r, text)
-        if clean:
-            urls = [trim_url(url) for url in urls]
-        return urls
-
-    find_urls = find_func
-
-
-@commands('title')
-@example('.title http://google.com', '[ Google ] - google.com')
+@module.commands('title')
+@module.example('.title http://google.com', '[ Google ] - google.com')
```

Comment:
Want to bring this link into the modern era while you're in here?

```suggestion
@module.example('.title https://www.google.com', '[ Google ] - www.google.com')
```